src/ConfigurationParser.cpp

```diff
@@ -95,6 +97,9 @@ Configuration ConfigurationParser::parse(
   configuration.usageCommand = extractCommand(p, USAGE_KEY);
 
   configuration.isDebugSet = getOrEmpty(p, DEBUG_KEY) == "true";
+
+  configuration.name = getOrEmpty(p, MODULE_NAME_KEY);
```

Comment:
The universe collapses 💥 
The string remains empty (which is not good because all isolators will write their context in the same file).
Quick failure is not a good idea I think, what could be nice would be to have a default name (based on isolator number for example) but I didn't manage to do this easily in ModuleFactory.cpp

Reply:
Let's discuss about fail fast approach then

Reply:
done